main.go

```diff
@@ -23,7 +23,7 @@ func main() {
 
 	ss := strings.Split(os.Args[1], "/")
 
-	if len(ss) > 2 {
+	if len(ss) > 1 {
```

Comment:
The original code is correct. Don't change this line.

---

master branch:

```console
$ dockertags wantedly/elasticsearch-snapshot-taker
latest
```

this PR:

```console
$ dockertags wantedly/elasticsearch-snapshot-taker
Unsupported image repository: wantedly
exit status 1
```

Reply:
Oh, I didn't found this bug because I (manually) tested only the situation like`dockertags mysql`

This change is for ECR URI is like `1234567890.dkr.ecr.ap-northeast-1.amazonaws.com/sample`.

I'll try fix this.

Reply:
How do you think about code below:

```suggestion
	if len(ss) > 2 || strings.HasSuffix(ss[0], "amazonaws.com") {
```

pros:  
- simple change

cons:
- little bit messy
  - It is better to divide this if-block into function to get repo and image

Reply:
I've fixed image name parser in #17. Please check it.

Reply:
Thanks! I applied the changes on this branch!